src/timer_esp32c3.rs

```diff
@@ -11,9 +13,9 @@ pub const TICKS_PER_SECOND: u64 = 16_000_000;
 #[cfg(debug_assertions)]
 const TIMER_DELAY: u32 = 8_000u32;
 #[cfg(not(debug_assertions))]
-const TIMER_DELAY: u32 = 500u32;
+const TIMER_DELAY: u32 = 3500u32;
```

Comment:
Should we make a note or open an issue regarding this?

Reply:
we could try to reduce interrupt latency for vectored interrupts - no idea if there are many options for improvement there

if there are some possible optimizations it probably makes sense to create an issue for it - if not it's probably not even worth to note it somewhere since "it's the way it is" IMHO

Reply:
Okay, just wasn't entirely sure what the deal with it was, thanks. Probably safe to just ignore for now, then.

Reply:
I opened https://github.com/esp-rs/esp-hal/issues/119, I don't have any concrete ideas but it is worth exploring regardless.